xbmc/addons/kodi-addon-dev-kit/include/kodi/versions.h

```diff
@@ -44,7 +44,7 @@
 #define ADDON_GLOBAL_VERSION_GENERAL_XML_ID           "kodi.binary.global.general"
 #define ADDON_GLOBAL_VERSION_GENERAL_DEPENDS          "General.h"
 
-#define ADDON_GLOBAL_VERSION_GUI                      "5.12.0"
+#define ADDON_GLOBAL_VERSION_GUI                      "5.12.1"
 #define ADDON_GLOBAL_VERSION_GUI_MIN                  "5.10.0"
```

Comment:
The min version has to be bumped for sure, as you removed a function from the public API.
I'm not sure if therefore a patch version bump is enough

Reply:
I bumped min version. not critical because this is only used by vnsi.